### ISSUE TITLE
Does Dosbox-X support 

### ISSUE THREAD

**Issue by WhyIsntTouhouBuntuRealYet** [joncampbell123/dosbox-x#2629]
**Please describe your question.**
A clear and concise description of what your question is. Ex. How do I [...]

**Comment by Wengier** [comment]
Is this the message you saw from the command?

`MSCDEX: Mounted subdirectory: limited support`

DOSBox-X does support CD Audio directly from the CD, but this feature may sometimes depend on your platform. Is the path `/media/QUAKE101` indeed referring to a CD? If not, the above message will appear. Also, the "-ioctl" option may not be supported in all platforms, even though it should work on most systems, such as my own Windows system.

**Comment by WhyIsntTouhouBuntuRealYet** [comment]
Yes, it refers to a CD. I have an original Quake CD, version 1.0.1.
Also, Dosbox slows down considerably whenever I use this command. Perhaps it's too taxing on the program.

I know it's not my Pi, because Quakespasm plays CD Audio perfectly. Maybe a future build can remedy this?

**Comment by Wengier** [comment]
I do not have TwisterOS to test myself, but the issue could be specific to this environment, or there is something in the said path which is not supported at this time. If I personally have such an environment I would be able to test it myself and see if there is anything to fix.

**Comment by rderooy** [comment]
As the message when mounting already hints at, it is a mounted **directory**, and therefore there will not be any CD Audio.

For there to be CD Audio, you should be a way point DOSBox-X to the CD device, not the mounted directory. This will normally be something like ``/dev/sr0`` (there is typically also a ``/dev/cdrom`` symlink). However that will not work, as there is no support in DOSBox-X to directly access a raw CD-ROM device on Linux.

The real solution is to convert the CD-ROM into a CUE/BIN image, and IMGMOUNT that.

**Comment by WhyIsntTouhouBuntuRealYet** [comment]
Is there a Dosbox build that CAN directly access the CD drive?

**Comment by Wengier** [comment]
@WhyIsntTouhouBuntuRealYet Yes, of course, although maybe not for specific platforms.

On Windows, a command like `mount c d:\ -t cdrom -usecd 0 -ioctl` should work just fine for CD drive D: for Windows builds.

On macOS, a command like `mount c /Volumes/cddrive -t cdrom -usecd 0 -ioctl` is supposed to work for CD drive /Volumes/cddrive too for macOS builds (just confirmed myself).

CD drives may be expected to work differently for different platforms.
